production.dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:lts-alpine as builder
+# Set the current working directory inside the container
+WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm ci
+COPY . .
+RUN npm run build
+
+# nginx state for serving content
+FROM nginx:1.21.6-alpine
+COPY nginx.conf /temp/prod.conf
+RUN envsubst /app < /temp/prod.conf > /etc/nginx/conf.d/default.conf
+# Set working directory to nginx asset directory
```

Comment:
This is stolen off circles, I would encourage you to consider if this is nessesary, and could instead use `serve`